MPF.Core/Modules/Redumper/Parameters.cs

```diff
@@ -1306,14 +1340,25 @@ protected override bool ValidateAndSetParameters(string? parameters)
                 if (!string.IsNullOrEmpty(stringValue))
                     SkipValue = stringValue;
 
-                // Skip
+                // Dump Write Offset
+                intValue = ProcessInt32Parameter(parts, FlagStrings.DumpWriteOffset, ref i);
+                if (!string.IsNullOrEmpty(stringValue))
```

Comment:
This and the following are invalid checks. Both should use the same checks that LBA Start and End are using.

Reply:
Why `!= Int32.MinValue` ?

Reply:
https://github.com/SabreTools/MPF/blob/65ad629ee0c90f77c925fc854b6bc653d4399943/MPF.Core/Modules/BaseParameters.cs#L755C17-L755C17

Reply:
Oh I see